source/DSCResources/DSC_CMSoftwareDistributionComponent/en-US/DSC_CMSoftwareDistributionComponent.strings.psd1

```diff
@@ -0,0 +1,14 @@
+ConvertFrom-StringData @'
+    RetrieveSettingValue  = Getting results for Configuration Manager accounts.
+    ComputerAccessAccount = Specifying both ComputerAccount and AccessAccount, these settings can not be specified together.
+    AccountsFalse         = Setting ClientComputerAccount to false and no access account is currently set or specified.
+    ParamIgnore           = AccessAccounts is specified, ignoring AccessAccountsToInclude and AccessAccountsToExclude for testing.
+    AccessAccountsInEx    = AccessAccountsToInclude and AccessAccountsToExclude contain the same member {0}.
+    AllAccountsRemoved    = All AccessAccounts would be removed causing the ClientComputerAccount to be set to true causing invalid configuration.
+    TestState             = Test-TargetResource compliance check returned: {0}.
+    ParamsError           = AccessAccounts and AccessAccountsToInclude or AccessAccountToExclude is specified remove AccessAccounts or the include or exclude setting.
+    ModifySetting         = {0} expected value: {1}, changing value.
+    AddingAccount         = AccessAccount is missing adding {0}.
```

Comment:
add a comma after missing. Below, add a comma after manager on line 12 and a comma after absent on line 13